integration/travis.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -ex
+
+sudo groupadd nobody
+sudo useradd hello
+
+sudo /sbin/stop runsvdir
+sudo sed -i -e 's;/usr/sbin/runsvdir-start;/usr/bin/runsvdir /var/service;g' /etc/init/runsvdir.conf
+sudo /sbin/start runsvdir
+sudo mkdir -p /etc/servicebuilder.d /var/service-stage /var/service
```

Comment:
yes. The default debian/ubuntu init script for runsvdir starts it under /etc/service instead of /var/service.